less/updateBar.less

```diff
@@ -31,11 +31,14 @@
   }
 
   .updateButton {
+    cursor: pointer;
```

Comment:
I don't think we want pointer's on buttons at all because it makes it look less native looking that way and less html looking.  Like for example we don't have a pointer on the back / forward buttons.

CC @bradleyrichter for thoughts.